hazelcast-hibernate5/src/main/java/com/hazelcast/hibernate/local/LocalRegionCache.java

```diff
@@ -87,12 +86,25 @@ public LocalRegionCache(final String name, final HazelcastInstance hazelcastInst
      */
     public LocalRegionCache(final String name, final HazelcastInstance hazelcastInstance,
                             final CacheDataDescription metadata, final boolean withTopic) {
+        this(name, hazelcastInstance, metadata, lookupMapConfig(name, hazelcastInstance), withTopic);
```

Comment:
existing constructor now delegates to the new one, using the new `lookupMapConfig` method